docs/2.7.0/docs/daml-finance/instruments/generic.rst

```diff
@@ -1,5 +1,162 @@
 .. Copyright (c) 2023 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
 .. SPDX-License-Identifier: Apache-2.0
 
-Generic Instrument
-##################
\ No newline at end of file
+How to use the Generic Instrument packages
```

Comment:
Here and I think in other places we say "Instrument packages" (plural). Are there actually multiple ones, or should this be singular?

Reply:
Each instrument is "distributed" across two packages (the interface and the implementation), hence I've been using plural.

I was actually planning of adding a table for each instrument enumerating the two packages, wdyt?

Reply:
Nah, I think it's good. thanks